test/functional/tests.go

```diff
@@ -9,6 +9,7 @@ var (
 		{Description: "A21 - Verify AWS maintenance and backup windows", Test: CROStrategyOverrideAWSResourceTest},
 		{Description: "A25 - Verify standalone RHMI VPC exists and is configured properly", Test: TestStandaloneVPCExists},
 		{Description: "F04 - Verify AWS s3 blob storage resources exist", Test: TestAWSs3BlobStorageResourcesExist},
+		{Description: "H24 - Verify selfmanaged Apicast", Test: TestSelfmanagedApicast},
```

Comment:
Does this test require to be ran on installations that have `clusterStorage` set to `false`?

Reply:
adding discussion from Chat:
Jim Fitzpatrick:
Currently you have the new test in the FUNCTIONAL_TESTS suite this only runs when clusterStorage is false. This would means it wont run in prow or if you run the tests locally with clusterStorage: true. When I was reading how the test is working I think this can run out side of the FUNCTIONAL_TESTS suite.

Valery Mogilevsky:
Thank you for reply Jim. Sorry for confusion. Actually there is another test file exists, that can be executed outside of test suite, it's located atls scripts/products/h24-verify-selfmanaged-apicast-and-custom-policy/
h24-verify-selfmanaged-apicast-and-custom-policy.go  test.sh  Both scripts functional/selfmanaged_apicast.go and h24-verify-selfmanaged-apicast-and-custom-policy.go are identical (except few minor things). I thought that it will be good meanwile to save h24 script, as it's easy for investigation

Reply:
Done
The test moved from functional to IDP based tests (test/common):
IDP_BASED_TESTS = []TestSuite{
    {
        []TestCase{
          .....
          {Description: "H24 - Verify selfmanaged Apicast", Test: TestSelfmanagedApicast},
	},
    .....